components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx

```diff
@@ -91,10 +110,18 @@ export const contactSalesCard = (contactSalesLink: any, isFreeTrial: boolean, tr
                     className='PrivateCloudCard__contactSales'
                     onClick={() => trackEvent('cloud_admin', 'click_contact_sales')}
                 >
-                    <FormattedMessage
-                        id='admin.billing.subscription.privateCloudCard.contactSales'
-                        defaultMessage='Contact Sales'
-                    />
+                    {subscriptionPlan !== 'CLOUD_STARTER' &&
+                        <FormattedMessage
+                            id='admin.billing.subscription.privateCloudCard.contactSales'
+                            defaultMessage='Contact Sales'
+                        />
+                    }
+                    {subscriptionPlan === 'CLOUD_STARTER' &&
+                        <FormattedMessage
+                            id='admin.billing.subscription.privateCloudCard.upgradeNow'
+                            defaultMessage='Upgrade Now'
+                        />
+                    }
```

Comment:
For this one which is actually "Upgrade Now", this should open the upgrade modal instead of contact sales right?

Reply:
I thought the same initially, but I was not sure how is the process of upgrade once you have the cloud_starter tier. Is it via the purchase modal?

Reply:
[MM-35835](https://mattermost.atlassian.net/browse/MM-35835) will cover the upgrade option and will be addressed outside of this PR - as discussed in today's triage